packages/create-react-router/index.ts

```diff
@@ -512,19 +512,11 @@ async function doneStep(ctx: Context) {
   await sleep(200);
 }
 
-type PackageManager = "npm" | "yarn" | "pnpm" | "bun";
-
-const packageManagerExecScript: Record<PackageManager, string> = {
```

Comment:
the create-react-router project doesn't seem to have init script support that remix did so i've refactored this a little to simplify.